NuGet/ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
```

Comment:
Should we call it `Microsoft.ChakraCore.Debugger` to align with the existing `Microsoft.ChakraCore` packages?

Reply:
On it...

Reply:
Done.